tools/baseimage/pkg/gce/scripts/scripts.go

```diff
@@ -29,7 +29,10 @@ var InstallNvidia string
 var FillAvailableDiskSpace string
 
 //go:embed install_cuttlefish_debs.sh
-var InstallCuttlefishPackages string
+var InstallCuttlefishDebs string
+
+//go:embed install_cuttlefish_container.sh
+var InstallCuttlefishContainer string
```

Comment:
the script is only loading the container image, can it be named "LoadCuttlefishContainerImage" instead?